### ISSUE TITLE
6D Rotation Representation For Unconstrained Head Pose Estimation. (arXiv:2202.12555v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#16159]
6D Rotation Representation For Unconstrained Head Pose Estimation. (arXiv:2202.12555v1 [cs.CV])<br>
https://ift.tt/r9C36Gx<br>
<p>In this paper, we present a method for unconstrained end-to-end head pose estimation. We address the problem of ambiguous rotation labels by introducing the rotation matrix formalism for our ground truth data and propose a continuous 6D rotation matrix representation for efficient and robust direct regression. This way, our method can learn the full rotation appearance which is contrary to previous approaches that restrict the pose prediction to a narrow-angle for satisfactory results. In addition, we propose a geodesic distance-based loss to penalize our network with respect to the SO(3) manifold geometry. Experiments on the public AFLW2000 and BIWI datasets demonstrate that our proposed method significantly outperforms other state-of-the-art methods by up to 20\%. We open-source our training and testing code along with our pre-trained models: https://ift.tt/kv9m7Zi.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
